### ISSUE TITLE
Share and fullscreen buttons are no longer available in version 5.4.3

### ISSUE THREAD

**Issue by Ferdinandgushcloud** [dimsemenov/PhotoSwipe#2100]
Hi All, 

Currently, I am considering upgrading from version 4.1.3 to 5.4.3. I have noticed that the current version does not include the share and fullscreen features

`<button class="pswp__button pswp__button--close" title="Close (Esc)"></button>`
  `<button class="pswp__button pswp__button--share" title="Share"></button>`
  `<button class="pswp__button pswp__button--fs" title="Toggle fullscreen"></button>`
  `<button class="pswp__button pswp__button--zoom" title="Zoom in/out"></button>`

I have reviewed the documentation, but perhaps I missed some information. I just want to confirm: does this latest version no longer provide the share and fullscreen features, as in version 4.1.3?

**Comment by dimsemenov** [comment]
Yes, that's correct. History API was removed as it's not reliable enough (too dependent on existing website routing and SEO/linking issues). And fullscreen feature was removed as it doubles existing behavior (e.g. click to open image and then click again to open it "more"). You can check this though for fullscreen https://photoswipe.com/native-fullscreen-on-open/

**Comment by Ferdinandgushcloud** [comment]
Thank you for the response, @dimsemenov . However, what about the 'Share' feature? Has it also been removed? As far as I understand from the documentation, we can now utilize the 'ui' feature as outlined in this documentation [https://photoswipe.com/adding-ui-elements/](https://photoswipe.com/adding-ui-elements/) to develop any custom code for it. Is that correct?

**Comment by arnowelzel** [comment]
For fullscreen support you can also add this plugin: https://github.com/arnowelzel/photoswipe-fullscreen
